src/main/java/org/opentripplanner/standalone/config/routerconfig/updaters/azure/SiriAzureETUpdaterConfig.java

```diff
@@ -1,4 +1,4 @@
-package org.opentripplanner.standalone.config.updaters.azure;
+package org.opentripplanner.standalone.config.routerconfig.updaters.azure;
```

Comment:
should we have the packages here according to the source (azure, google, aws) or per type (siri, gtfsrt, etc)?

Reply:
I think we should group by feature/functionality then tech. I don´t want to change this in this PR.